packages/video_player/lib/video_player.dart

```diff
@@ -87,10 +135,18 @@ class VideoPlayerValue {
   /// Is null when [initialized] is false.
   final Size size;
 
+  /// False when [duration] is null, true otherwise.
   bool get initialized => duration != null;
+
+  /// Whether or not [errorDescription] is null.
   bool get hasError => errorDescription != null;
+
+  /// Returns [size.width] / [size.height] when size is non-null, or `1.0.` when
```

Comment:
ditto as `initialized`

Reply:
I think in this specific case (unlike the other two) this is actually explaining what the getter does in a way that's hard to implify. An aspect ratio really is either width / height or height / width, and I think it's useful to know what happens in strange edge cases where there aren't any dimensions loaded yet.